src/sinks/datadog/metrics/encoder.rs

```diff
@@ -738,6 +760,47 @@ mod tests {
         }
     }
 
+    fn encode_sketches_normal<B>(
+        metrics: &[Metric],
+        default_namespace: &Option<Arc<str>>,
+        log_schema: &'static LogSchema,
+        buf: &mut B,
+    ) where
+        B: BufMut,
+    {
+        let mut sketches = Vec::new();
+        for metric in metrics {
+            match metric.value() {
+                MetricValue::Sketch { sketch } => match sketch {
```

Comment:
This looks like a `let MetricValue::Sketch { sketch } = metric.value() else { panic!() };`

Reply:
Yes, yes it does.

I haven't actually taken advantage of let-else yet. 😄

Reply:
Addressed in f533dd7ef.